app/services/reporter.rb

```diff
@@ -6,38 +6,76 @@
 # run queries and produce reports from the results, for consumption
 # by preservation catalog maintainers
 class Reporter
-  # @param [String] the name of the storage root for which druids should be listed
+  attr_reader :storage_root, :druids
+
+  # @param [String] the name of the storage root to initialize for reporter
+  # @return [Reporter] the reporter
+  def initialize(params)
+    @storage_root = MoabStorageRoot.find_by!(name: params[:storage_root_name])
+    @druids = []
```

Comment:
I wonder if you'd be open to `moab_storage_root_druid_list` returning the array of hashes it's generating, and assigning that to the value of `@druids` rather than setting the ivar to a blank array and then mutating it?